src/gs-details-page.c

```diff
@@ -2265,6 +2265,16 @@ gs_details_page_network_available_notify_cb (GsPluginLoader *plugin_loader,
 	gs_details_page_refresh_reviews (self);
 }
 
+static void
+gs_details_page_key_pressed_cb (GtkWidget *widget, GdkEventKey *key, GsDetailsPage *self)
+{
+	g_return_if_fail (key != NULL);
```

Comment:
I don't think this condition here is needed; `g_return_if_fail()` are used for programmer errors, but if we arrive here with `key == NULL` it's a bug in gnome-software and nothing that the caller of this function can do about.